config.yaml

```diff
@@ -1,8 +1,8 @@
 default:
  log_file: "/var/log/host-os/builds.log"
  packages_repos_target_path: "/var/lib/host-os/repositories"
- build_versions_repository_url: "https://github.com/open-power-host-os/versions.git"
- build_version: 'master'
+ packages_metadata_repo_url: "https://github.com/open-power-host-os/versions.git"
```

Comment:
I think "packages metadata" is too generic. The new user will not have any clue what is this metadata. By having "versions" in the name, he at least knows what repo it refers to. It also makes explicit *only* the versions repo (or a fork) can be used here.
Having to write "packages metadata" in every variable and documentation, in my opinion, just adds extra words and complexity with no gain of value.